src/main/java/csv/WriteCSV.java

```diff
@@ -0,0 +1,36 @@
+package main.java.csv;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriteCSV {
+
+    private final static String PEOPLE_DATA = "people.csv";
+
+    public static void main(String[] args) {
+        PersonCSV header = new PersonCSV("Name", "Age", "City");
```

Comment:
Interesting solution, but it is too implicit.
Maybe it is better to introduce a separate method, that will generate the header CSV row...